src/content/platform-integration/platform-channels.md

```diff
@@ -894,7 +894,11 @@ And replace with the following:
 You should now be able to run the application on Windows.
 If your device doesn't have a battery,
 it displays 'Battery level not available'.
-  
+:::note
+To ensure that the changes take effect, recompile the project.
+Otherwise, you might encounter the following error: **Failed to get 
+battery level: Memory leak.**
```

Comment:
I don't see how it's possible to get this error message string from the provided example code, so this needs further debugging with repro steps; changing docs is premature (especially since the step the new text is recommending should not be necessary).

Reply:
> I don't see how it's possible to get this error message string from the provided example code, so this needs further debugging with repro steps; changing docs is premature (especially since the step the new text is recommending should not be necessary).

Should i open an issue with repro steps first ?